apps/backend/src/drinks/entities/drink.entity.ts

```diff
@@ -0,0 +1,44 @@
+import { DrinkType } from '@prisma/client';
+import { IsBoolean, IsDateString, IsEnum, IsNotEmpty, IsOptional, IsString, IsUUID, Min } from 'class-validator';
+import { IsFloat } from 'src/util/is-float.validator';
+
+export class Drink {
+  @IsUUID()
+  id: string;
+
+  /**
+   * Name of the drink
+   * @example Soproni meggy
```

Comment:
The swagger default values don't work, the example string values should be between apostrophes.
  /**
   * Name of the drink
   * @example 'Soproni meggy'
   */

Reply:
good catch!